config/deploy/production.rb

```diff
@@ -1,5 +1,5 @@
 set :stage, :production
 set :rails_env, 'production'
 
-server ENV['SERVER'], user: 'deploy', roles: [:web, :app, :db]
+server ENV['SERVER'], user: 'adrl', roles: [:web, :app, :db]
```

Comment:
You'll want the resque_pool role here as well - see https://github.com/curationexperts/alexandria-v2/blob/master/config/deploy/aws.rb#L8